### ISSUE TITLE
Collider action does not detect Character Controller collider

### ISSUE THREAD

**Issue by PhyresiCompany** [FlaxEngine/FlaxEngine#1200]
As shown in this example, the action is fired when my Mushroom collides with my BoxCollider, but it doesn't fire when my Character Controller collides with my BoxCollider

- My Player is a CharacterController + AnimatedModel
- My Mushroom is a RigidBody + StaticModel + MeshCollider

I need the action to detect my CharacterController so I can add my logic from there

https://github.com/FlaxEngine/FlaxEngine/assets/101017436/f2afcd31-4c77-4cf1-b070-b1895109305e

**Script**
```csharp
public class Col : Script
{
    public Collider collider;

    public override void OnEnable()
    {
        collider.CollisionEnter += Collider_CollisionEnter;
    }

    public override void OnDisable()
    {
        collider.CollisionEnter -= Collider_CollisionEnter;
    }

    private void Collider_CollisionEnter(Collision obj)
    {
        Debug.Log("Yes Collision");
    }
}
```

**Comment by mafiesto4** [comment]
Thanks for reporting this. It looks like a very important thing to fix.

**Comment by mafiesto4** [comment]
Fixed in 694f8fbaf3c86a2a0a13869d5b532879885366d1
